composeApp/src/main/kotlin/io/github/couchtracker/tmdb/TmdbLanguages.kt

```diff
@@ -66,6 +83,31 @@ private fun rememberSystemTmdbLanguages(): TmdbLanguages {
 
 @Composable
 fun TmdbLanguages?.orDefault(): TmdbLanguages {
-    val systemTmdbLanguages = rememberSystemTmdbLanguages()
+    val systemTmdbLanguages = LocalConfiguration.currentLocales.toTmdbLanguages()
     return this ?: systemTmdbLanguages
 }
+
+@OptIn(ExperimentalCoroutinesApi::class)
+fun Flow<TmdbLanguages?>.orDefault(): Flow<TmdbLanguages> {
```

Comment:
I don't like the duplication of having effectively the same thing: one in compose, one with flows. However, the `orDefault` composable is needed in the settings screen. Should we try to remove it?